### PR TITLE
Fixed: Update PR labeller

### LINKED ISSUE #766: PR labeled fails on PRs from fork

### PR DESCRIPTION
**Pull request by ahankinson**
Updates the PR labeller to run on the PR target, rather than the pull request, to get around a permissions issue running on a fork.

Fixes #766

### CODE REVIEW

**Comment by ahankinson** [comment]
Tagging @musicEnfanthen as well

**Comment by musicEnfanthen** [comment]
Thanks @ahankinson for the PR. Looks good. Indeed, this labelling action seems to be one of the rare use cases where the official recommendation is to use pull_request_target.

BTW: Do we want to run the GH action only on music-encoding or also on all forks (i.e. all the forks get the labels, too). If only the former then we might want to include something like

`if: github.repository_owner == 'music-encoding'`

at line [22](https://github.com/music-encoding/music-encoding/blob/6f9ba917117b0757520c100f831f9796bebf525a/.github/workflows/pr-labeler.yml#L22).

**Comment by ahankinson** [comment]
As far as I can tell, this will only run on forks if someone is targeting a pull request to that fork... that's what is meant by the 'target'. So I think the `github.repository_owner == 'music-encoding'` is implied, unless someone is maintaining a parallel MEI repo.

**Comment by musicEnfanthen** [comment]
Can we exclude that some forks are working with Pull Requests to their own repos at their covenience? Or to put it the other way round: inserting the if clause would ensure that the gh action is never ever executed outside the organisational main repo. Cf. https://github.com/NixOS/nixpkgs/pull/113077/files#r575748234

(Actually, that is even more urgent for the the [deploy.yml](https://github.com/music-encoding/music-encoding/blob/develop/.github/workflows/deploy.yml))

**Comment by ahankinson** [comment]
Added; let me know if it looks ok?

File: .github/workflows/pr-labeler.yml [hunk 1]
```diff
@@ -7,19 +7,17 @@
...
-#  workflow_dispatch:
 
 jobs:
   label:
-
     runs-on: ubuntu-latest
-
+    if: github.repository_owner == 'music-encoding'
     steps:
     - uses: actions/labeler@v2
```
Comment (musicEnfanthen) [inline .github/workflows/pr-labeler.yml]:
```suggestion
    - uses: actions/labeler@v3
```

**Review by musicEnfanthen — APPROVED** [review]

**Comment by musicEnfanthen** [comment]
Yes, thank you! (Suggestion to update labeler version to v3)

**Comment by bwbohl** [comment]
dear both, thanks for working on this and spotting the problem esp. with the deploy action! Could you submit a PR for that one, too?

**Review by bwbohl — APPROVED** [review]

**Comment by bwbohl** [comment]
this ready to merge?

**Comment by ahankinson** [comment]
Good from me.

**Pull request merged**